### DISCUSSION TITLE
FUNDME.T.SOL DOUBT

### DISCUSSION THREAD

**Discussion by TechnicallyKiller** [Cyfrin/foundry-full-course-cu#2760] [Q&A]
Uhm just a confusion do i need to setup in fundme.t.sol after i made the helperconfig file and also done with importing it in deploy.s.sol  now its gonna take up the address on block.chainID
![image](https://github.com/user-attachments/assets/db275f11-6fa8-4ce1-b758-3a90c0ab59cf)
![image](https://github.com/user-attachments/assets/c00a7c8f-7b66-4340-9214-bca69cdedbc1)

**Comment by EngrPips** [comment]
Hello @TechnicallyKiller, I don't understand your question. Please make it clearer to me. I also want to mention that if you follow Patrick closely, you will figure out everything you have to do.
